robotics-4-all/dFlow
# Access Control

## Code review checklist

- [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR)
- [ ] Title summarizes what is changing
- [ ] Commit messages are meaningful (see [this][commit messages] for details)
- [ ] Tests have been included and/or updated
- [ ] Docstrings have been included and/or updated, as appropriate
- [ ] Standalone docs have been updated accordingly
- [ ] Changelog(s) has/have been updated, as needed (see `CHANGELOG.md`, no need
      to update for typo fixes and such).


[commit messages]: https://chris.beams.io/posts/git-commit/

Done:
1) Add access control with two policy options (ActionGroup-based, Action-Based)
2) Add connectors for slack and Telegram

Open Issues:
1) SetFormSlot and SetGlobalSlot action types assigned on generator.py dont match with SetSlot action type used in actions.py.jinja
2) Try-catch needed in actions.py.jinja when "request.(get,post,put, etc..)" is used, because it throws an error when the domain is unreachable
